### PR TITLE
Dusty aladdin patch 1

### PR DESCRIPTION
**Pull request by DustyAladdin**

### CODE REVIEW

File: source/guide/index.md [hunk 1]
```diff
@@ -91,13 +91,15 @@ It will show you how many units of each type you have in your main fleet. Below
 
 In a galaxy that allows this feature you can buy resources that you need or sell them for gold. Sometimes there is a market strike where the market is shut down for a period of time. Bids to buy resources will remain on the market until they are filled by another player selling the desired amount of resources. Gold stored in bids on the market will not decay in the same way bureaucracy decays 0.5% of saved gold. Because of this, it is possible to save large amounts of gold on the market in bids that will not be filled and cancel the order later so that losses from the 10% cancellation fee. Resources stored in the market as open bids still decay at the same 0.5% as resources saved normally. 
 
+The market functions similarly to a stock market, so large gains are possible by buying low and selling high if there are enough active players using it.
```
Comment (ImperialConflict) [inline source/guide/index.md:94]: can we remove the "if there are enough active players using it." part?

File: source/guide/index.md [hunk 2]
```diff
@@ -117,37 +119,47 @@ This page is where you do your spells. It tells you how much it will cost in oct
...
 
 ##### Personal Log
 
 You can make notes or keep information here for your own personal reference.
 
 ##### Rankings
 
-This link will take you to the ranking page to show you where your family and your empires are compared to the other families in the galaxy. There are 3 different ranking systems: Score, Size & Networth.
+This link will take you to the ranking page to show you where your family and your empires are compared to the other families in the galaxy. There are 3 different ranking systems: Score, Size & Networth. Size ranks solely on number of planets, networth ranks solely on networth, and score ranks based on a combined size/networth taking the entire history of the round into account. Players generally take size ranking at the conclusion of the round to be by far the most important ranking.
```
Comment (ImperialConflict) [inline source/guide/index.md:130]: > Players generally take size ranking at the conclusion of the round to be by far the most important ranking.

This one is somewhat debateable.  Could you scrap this line?

File: source/guide/index.md [hunk 3]
```diff
@@ -117,37 +119,47 @@ This page is where you do your spells. It tells you how much it will cost in oct
...
-## Your Family and Home System
+## Your Family and Player Roles
 
 After you decide which galaxy you will want to join, you have your own planet in a system called your home system.  In this system are other empires and their home planets.  These empires are your family members, and this is your family.  This is your team.
 
 You can view your family's page by clicking the "Family" link on the left sidebar and choosing "Overview".  This will show you all the other empires that are in your family, along with a subnavigation with other relevant links.
 
 One of the first things you want to do is post in your family's forum to introduce yourself.  A lot of good planning happens here and it's a great place to offer help and/or ask questions.
 
+Due to race bonuses lying in different areas of expertise, players in a family will divide themselves into different roles to cover all of the family's needs as efficiently as possible. The three main roles that the vast majority of players in a galaxy will fall into are bankers, resourcers, and attackers.
```
Comment (ImperialConflict) [inline source/guide/index.md:146]: This section (and below) is a fantastic addition.  Great work!

**Review by ghost — CHANGES_REQUESTED** [review]
Great work.  There's a couple of adjustments to be made though.  Thanks.

**Review by ghost — APPROVED** [review]
Great work, thank you.

**Pull request merged**